CorsixTH/Lua/app.lua

```diff
@@ -313,6 +313,39 @@ function App:init()
   return true
 end
 
+---
+-- Initialises a Lua DBGp client and
+-- connects it to a DBGp server so that
+-- it can debug lua scripts at runtime.
+-- @return error_message
+---
+function App:connectDebugger()
+  local error_message = is_file_readable("../Lua/debugger.lua")
```

Comment:
I would prefer to find the absolute path using something like debug.getinfo(1, "S").source:sub(2, -8) (gets this file's path, see other places in the code) as the current approach relies on a specific current working directory. When running through Visual Studio the cwd is different from when you run the executable itself.